lib/metrics/index.go

```diff
@@ -0,0 +1,118 @@
+			"WriteStorageDurationNs":  "The total delay of the entire writing process on the ts-store node",
+			"WriteGetTokenDurationNs": "On the ts-sotre node, when writing data, the cumulative delay in the flow control logic waiting for resource allocation (ns)",
+			"WriteIndexDurationNs":    "On the ts-store node, the cumulative delay in creating an index when writing data (ns)",
+			"WriteRowsDurationNs":     "On the ts-store node, the cumulative delay of writing to the cache (the granularity is coarser than WriteMstInfoNs)",
+			"WriteWalDurationNs":      "Accumulated delay of writing to WAL (ns)",
+			"WriteRowsCount":          "On the ts-store node, the total number of rows of data written",
+		},
+		Labels: []string{"app", "hostname"},
+	},
+	"io": {
+		HelpMap: map[string]string{
+			"snapshotBytes": "Snapshot file size (cumulative, calculated in Byte)",
+		},
+		Labels: []string{"app", "hostname"},
+	},
+	"executor": {
+		HelpMap: map[string]string{
+			"exec_run_time_count": "The total number of executor executions of the query plan",
+			"exec_abort_count":    "The total number of query plan executor terminations",
+			"exec_failed_count":   "The total number of executor execution failures for the query plan",
+			"exec_timeout_count":  "The total number of executor timeouts for the query plan",
+			"source_rows_sum":     "The total number of rows of entry data when the executor is executed",
+			"sink_rows_sum":       "The total number of rows of export data when the executor is executed",
+		},
+		Labels: []string{"app", "hostname"},
+	},
+	"system": {
+		HelpMap: map[string]string{
+			"CpuUsage":     "cpu utilization",
+			"MemUsage":     "Node memory usage",
+			"CpuNum":       "Number of node CPU cores",
+			"MemSize":      "Node memory capacity",
+			"DiskSize":     "Node main disk capacity",
+			"IndexUsed":    "Index data uses disk capacity",
+			"AuxDiskSize":  "Auxiliary disk capacity. WAL files are generally stored on different disks to improve data security. See the ts-monitor.conf aux-disk-path configuration item.",
+			"StoreStatus":  "Store process status, 1 (running) 0 (killed)",
+			"DiskUsage":    "Node primary disk space usage",
+			"AuxDiskUsage": "Secondary disk space usage",
+			"MemInUse":     "Used memory size",
+			"SqlStatus":    "SQL process status, 1 (running) 0 (killed)",
+			"MetaStatus":   "Meta process status, 1 (running) 0 (killed)",
+			"Uptime":       "Node running time (s)",
+			"NumGC":        "Number of GCs completed",
+		},
+		Labels: []string{"app", "hostname"},
+	},
+	"runtime": {
+		HelpMap: map[string]string{
+			"NumGC":        "Number of GCs completed",
+			"NumGoroutine": "Number of Goroutines generated",
+		},
+		Labels: []string{"app", "hostname"},
+	},
+	"spdy": {
+		HelpMap: map[string]string{
+			"connTotal": "Total number of connections (internal)",
+		},
+		Labels: []string{"app", "hostname", "link", "remote_addr"},
+	},
+	"measurement_metric": {
```

Comment:
How to cancel the metrics when the `measurement` or `database` is deleted?

Reply:
If this database does not exist, it will be ignored.